test/Microsoft.AspNetCore.Server.KestrelTests/BadHttpRequestTests.cs

```diff
@@ -52,6 +52,29 @@ public class BadHttpRequestTests
         [InlineData("} / HTTP/1.0\r\n")]
         [InlineData("get@ / HTTP/1.0\r\n")]
         [InlineData("post= / HTTP/1.0\r\n")]
+        [InlineData("GET http:// HTTP/1.1\r\n")]
+        [InlineData("GET http:/// HTTP/1.1\r\n")]
+        [InlineData("GET https:// HTTP/1.1\r\n")]
+        [InlineData("GET http://// HTTP/1.1\r\n")]
+        [InlineData("GET http://:80 HTTP/1.1\r\n")]
+        [InlineData("GET http://:80/abc HTTP/1.1\r\n")]
+        [InlineData("GET http://user@ HTTP/1.1\r\n")]
+        [InlineData("GET http://user@/abc HTTP/1.1\r\n")]
+        [InlineData("GET http://abc%20xyz/abc HTTP/1.1\r\n")]
+        [InlineData("GET http://%20/abc?query=%0A HTTP/1.1\r\n")]
+        // TODO invalidate requests that don't meet the HTTP spec. See https://github.com/aspnet/KestrelHttpServer/issues/1279
+        //[InlineData("GET otherscheme://host/ HTTP/1.1\r\n")]
+        //[InlineData("GET ws://host/ HTTP/1.1\r\n")]
+        //[InlineData("GET wss://host/ HTTP/1.1\r\n")]
```

Comment:
If there is an absolute-form request with a non HTTP(S) scheme, does Path become string.Empty since it doesn't start with '/' while not being rejected?